src/EntityFrameworkCore.Generator.Core/ModelGenerator.cs

```diff
@@ -540,10 +540,13 @@ private string EntityName(string name)
         }
 
 
-        private string ToClassName(string name)
+        private string ToClassName(string tableName, string tableSchema)
         {
-            name = EntityName(name);
-            string legalName = ToLegalName(name);
+            var className = tableName;
```

Comment:
We lost the EntityName() logic.  Should tableName name pass through EntityName() or the result after adding schema?

```c#
var className = EntityName(tableName);
```

Reply:
might have a harder time converting tense with schema prefixed. Table name more likely to be word.  So I'd think convert the table name, then prefix the schema.

Reply:
Added EntityName again